src/utils/convertData.js

```diff
@@ -0,0 +1,54 @@
+import dayjs from 'dayjs';
+import isBetween from 'dayjs/plugin/isBetween';
+import config from '../constants/config';
+
+const { days } = config;
+dayjs.extend(isBetween);
+
+const getTime = (date) => {
+  const time = date.set('m', 0);
+  return time.format('H');
+};
+
+const groupByDay = (posts) => {
+  const results = {};
+  // eslint-disable-next-line no-restricted-syntax
+  for (const post of posts) {
+    const date = dayjs.unix(post.created_utc);
+    const day = days[date.get('d')];
+    const time = getTime(date);
+
+    if (!results[day] || !results[day][time]) {
+      results[day] = { ...results[day], [time]: [] };
+    }
+
+    results[day][time].push(post);
+  }
+
+  return results;
+};
+
+const sort2 = (posts) => {
+  const sortedPosts = { ...posts };
+  Object.keys(sortedPosts).sort((a, b) => a - b);
```

Comment:
This is most probably not having any effect, since it's sorting the array that is returned by Object.keys. What are trying to sort here?

Reply:
I was trying to sort hours from 0 to 23, but I don't think i need this